pkg/oidcprovider/client.go

```diff
@@ -68,7 +69,10 @@ func (c client) IDTokenUserinfoClaimsAssertion() bool {
 // IsScopeAllowed implements op.Client.
 func (c client) IsScopeAllowed(scope string) bool {
 	slog.Warn("oidcprovider.client.IsScopeAllowed", "scope", scope)
-	panic("unimplemented")
+	if strings.HasPrefix(scope, "pls_") {
+		return true
+	}
+	return false
```

Comment:
This is just `return cond`; you probably did this for better extensibility in the future if we want to add more scopes, but honestly I don't know if it's worth it.

Up to you, if you have strong opinions.

Reply:
```java
if (true) {
  return true;
} else {
  return false;
}
```
is my favorite